util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -97,15 +97,19 @@ public static ApiClient defaultClient() throws IOException {
    *     cannot be read.
    */
   public static ClientBuilder standard() throws IOException {
-    return standard(true);
+    return standard(true, null);
   }
 
-  public static ClientBuilder standard(boolean persistConfig) throws IOException {
+  public static ClientBuilder standard(String context) throws IOException {
+    return standard(true, context);
+  }
+
+  public static ClientBuilder standard(boolean persistConfig, String context) throws IOException {
```

Comment:
Rather than change this signature, I think we should add a `context(String context)` method to `ClientBuilder`

Reply:
Done.